src/components/mods/Level100MOptions.ts

```diff
@@ -0,0 +1,20 @@
+import { Level100Mod } from "./Level100Mod";
+import { MOptions } from "./MOptions";
+
+export class Level100MOptions extends MOptions {
+    public constructor (name: string) {
+        super(name);
+    }
+    public readIn(): void {
```

Comment:
Why is this a member of `Level100MOptions`. Shouldn't something that generic be in `MOptions`?